app/src/main/java/com/aerogear/androidshowcase/features/documentation/DocumentUrl.java

```diff
@@ -4,14 +4,13 @@
  * This class enumerates the documentation pages we have.
  */
 public enum DocumentUrl {
-    PUSH("https://docs.aerogear.org/aerogear/latest/showcase/push.html"),
-    IDENTITY_MANAGEMENT("https://docs.aerogear.org/aerogear/latest/showcase/idm.html"),
-    DEVICE_SECURITY("https://docs.aerogear.org/aerogear/latest/showcase/device-security.html"),
-    METRICS("https://docs.aerogear.org/aerogear/latest/showcase/metrics.html"),
-    RUNTIME("https://docs.aerogear.org/aerogear/latest/showcase/runtime.html"),
-    IDENTITY_MANAGEMENT_SSO("https://docs.aerogear.org/aerogear/latest/keycloak/index.html?sso=1"),
-    NOTES_SERVICE("https://github.com/feedhenry/mobile-security/tree/master/projects/api-server"),
-    SELF_SIGNED_DOCS("https://docs.aerogear.org/aerogear/latest/getting-started.html#using-self-signed-certificates-in-mobile-apps");
```

Comment:
This seems to be used here https://github.com/aerogear/android-showcase-template/blob/115652bf457fe69e26864c885829364da8a9dfe5/app/src/main/java/com/aerogear/androidshowcase/navigation/Navigator.java#L239

Reply:
Fixed